app/src/main/java/com/sunlightlabs/android/congress/fragments/BillInfoFragment.java

```diff
@@ -249,60 +215,19 @@ public void addBillTimeline() {
 		if (bill.introduced_on != null)
 			addTimelinePiece(inner, "Introduced on", bill.introduced_on.getTime());
 		
-		String house_passage_result = bill.house_passage_result;
-		long house_passage_result_at = bill.house_passage_result_at == null ? 0 : bill.house_passage_result_at.getTime();
-		if (house_passage_result != null && house_passage_result_at > 0) {
-			if (house_passage_result.equals("pass"))
-				addTimelinePiece(inner, "Passed the House on", house_passage_result_at);
-			else if (house_passage_result.equals("fail"))
-				addTimelinePiece(inner, "Failed the House on", house_passage_result_at);
-		}
+		long house_passage_result_at = bill.house_passage_result_on == null ? 0 : bill.house_passage_result_on.getTime();
+		if (house_passage_result_at > 0)
+			addTimelinePiece(inner, "Passed the House on", house_passage_result_at);
```

Comment:
We now don't show veto override or cloture timeline moments (these aren't in the PP API), and House and Senate passage results are only shown when passage is successful.